app/data/test_household_question.json

```diff
@@ -82,11 +81,87 @@
+                                               "People who are temporarily outside the UK for <b>less than 12 months</b>",
+                                               "People staying temporarily who usually live in the UK but do not have another UK address, for example relatives, friends",
+                                               "Other people who usually live here, including anyone temporarily away from home "
+                                           ]
+                                      }
+                                  ],
+                                  "answers": [
+                                        {
+                                            "guidance": "",
+                                            "id": "household-composition-add-another",
+                                            "label": "",
+                                            "mandatory": true,
+                                            "options": [
+                                                {
+                                                    "label": "Yes",
+                                                    "value": "Yes"
+                                                },
+                                                {
+                                                    "label": "No - I need to add another person",
+                                                    "value": "No"
+                                                }
+                                            ],
+                                            "q_code": "4",
+                                            "type": "Radio",
+                                            "validation": {
+                                                "messages": {}
+                                            }
+                                        }
+                                  ]
+
+                                }
+                            ],
+                            "title": "Who lives here?",
+                            "description": "<h2 class='neptune'>Your household includes:</h2> {{ [answers.first_name, answers.middle_names, answers.last_name]|format_household_summary }}",
+                            "validation": []
+                        }],
+                        "routing_rules":[
+                            {
+                                "goto": {
+                                    "id" : "household-composition",
+                                    "when": {
+                                        "id" : "household-composition-add-another",
+                                        "condition": "equals",
+                                        "value":"No"
+                                    }
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "id": "summary",
```

Comment:
I think it makes sense in the context of the question, if you have entered everyone go to the next section which in the test schema is the summary